default-config.json

```diff
@@ -50,20 +50,30 @@
       "alias": "Cadence play "
     }
   ],
+  "commandFriendlyNames": {
+    "play": "play",
+    "stop": "stop",
+    "help": "help",
+    "helpTopic": "help",
+    "nowplaying": "now playing",
+    "search": "search ",
+    "request": "request ",
+    "library": "library"
+  },
   "helpTopics": {
     "help": {
       "internalKey": "help",
       "subtitle": "The help command provides information to assist with the usage of Cadence.",
       "lines": [
         "It can be used in one of two ways:",
-        "    1. Without any argument (`%help`), it will generate a summary of commands and a brief snippet of their purpose.",
-        "    2. Provided the name of a command (`%helpTopic help`), it will provide details regarding the specific usage of that command."
+        "    1. Without any argument (`%friendly:help`), it will generate a summary of commands and a brief snippet of their purpose.",
```

Comment:
This begs the question of when we should use friendly name vs canonical (an invocation)

So here, are we saying without any argument as in help, or are we saying "if you run it without any argument by sending the message `Cadence help`"?

Adding the friendly name creates the writing style concern of when you write a friendly name to refer to the command, and when you write the invocation of the command - For example, I might write "%friendly:search results will be saved for a future use of %request to select a song."

Not saying this is necessarily wrong, mind you - but we should try to avoid letting ourselves mindlessly use one form or the other, now that you're giving us the freedom of using either.

Reply:
Oh yeah, I wasn't thinking here and blindly changed all the canonical names into friendly names. It seems these should all be left as canonical names except the %nowplaying command.

Reply:
That sounds good to me!